### ISSUE TITLE
the problem of partition producer number 

### ISSUE THREAD

**Issue by yarthur1** [apache/pulsar-client-go#327]
#### Expected behavior

After a bundle successfully moved from broker A to broker B , goclient log show that partition producer has reconnected new broker, but goclient has lots of warn log like"level=warning msg="Got unexpected send receipt for message: ledgerId:87806 entryId:10433 "...". Grafana dashboard show the number of producer have some change,and eventually is not correct

broker log
broker A unload bundle
![image](https://user-images.githubusercontent.com/30923528/88007314-52758380-cb40-11ea-9829-d100be04dac4.png)

![image](https://user-images.githubusercontent.com/30923528/88007212-088c9d80-cb40-11ea-8086-100cefa1a439.png)

broker B load bundle
![image](https://user-images.githubusercontent.com/30923528/88007513-b7c97480-cb40-11ea-9180-faf4898d85c6.png)

![image](https://user-images.githubusercontent.com/30923528/88007442-923c6b00-cb40-11ea-84f8-832de9f428f1.png)

Go client log
![image](https://user-images.githubusercontent.com/30923528/88007978-bc425d00-cb41-11ea-9163-2bc02dda1932.png)
![image](https://user-images.githubusercontent.com/30923528/88007996-c9f7e280-cb41-11ea-8ad5-82ce0f7d2011.png)

grafana 
![image](https://user-images.githubusercontent.com/30923528/88008335-a5503a80-cb42-11ea-900e-9832aa004338.png)

#### System configuration
Pulsar version: 2.6.0
Go client version: 0.1.0

**Comment by sijie** [comment]
@yarthur1 I don't think it is a problem on either client or broker side. This sounds like a grafana problem. Have you checked the query to understand if it is a metric problem?

**Comment by yarthur1** [comment]
> @yarthur1 I don't think it is a problem on either client or broker side. This sounds like a grafana problem. Have you checked the query to understand if it is a metric problem?

But it lead to send rate slow, I have add timeout check on Go client send method to test and find lots of timeout

**Comment by LvBay** [comment]
@yarthur1 Did you use the same client instance to create both  consumers and a producers?
I have encountered a similar problem and have submitted a PR to fix it, hope it can help you.

**Comment by yarthur1** [comment]
> @yarthur1 Did you use the same client instance to create both consumers and a producers?
> I have encountered a similar problem and have submitted a [PR](https://github.com/apache/pulsar-client-go/pull/326#pullrequestreview-454602379) to fix it, hope it can help you.

yes,I  use the same client instance. thanks,I will view it
